packages/ui-extensions/src/surfaces/admin/components/shared.d.ts

```diff
@@ -2430,6 +2444,16 @@ interface DateFieldProps$1
       | 'onViewChange'
     >,
     AutocompleteProps<DateAutocompleteField> {
+  /**
+   * Callback when the user makes any changes in the field.
+   * Also triggered when a date is selected using the date picker popup before `onChange`.
+   */
+  onInput?: (event: Event) => void;
```

Comment:
I don't really have a ton of context into how ui-extensions works, tbh. But just making sure that these were added to the interfaces, but I don't see any code that handles these in this PR - was this simply just getting the interfaces updated and the code was already there?